docs/src/main/sphinx/admin/properties-query-management.rst

```diff
@@ -2,6 +2,18 @@
 Query management properties
 ===========================
 
+``query.execution-policy``
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-string`
+* **Default value:** ``phased``
+* **Session property:** ``execution_policy``
+
+The execution policy improves the wall time for query processing with the
+``phased`` scheduling policy. To restore previous previous policy behavior, set
+the ``query.execution-policy`` configuration property to ``legacy-phased`` or
```

Comment:
Do legacy-phased and all-at-once do the same thing? This reads like they do.

Reply:
No. Both are different. The older default was `all-at-once`.

`legacy-phased` earlier used to be called `phased` so to make space for the new one the policy was renamed.

cc: @sopel39 to correct if I'm wrong.